Source/WTF/wtf/cocoa/VectorCocoa.h

```diff
@@ -113,13 +113,13 @@ template<typename MapFunctionType> Vector<typename std::invoke_result_t<MapFunct
     return vector;
 }
 
-inline Vector<uint8_t> toVector(NSData *data)
+inline Vector<uint8_t> vector(NSData *data)
 {
-    return toSpan(data);
+    return span(data);
```

Comment:
In some call sites I see us doing `return { span() }` but here we just `return span()` without the braces.